Exercitii/numele-meu.js

```diff
@@ -0,0 +1,10 @@
+console.log("EXERCITIUL 4")
+
+let Nume = "Andrei"
+
+let Prenume = "Dragomir"
+
+
+console.log("Numele meu este:")
```

Comment:
Aici problema nu este completa, deoareace se va afisa doar textul "Numele meu este" iar cerinta indica ca trebuie si valoarea din variabile. Solutia in acest caz ar fi
```js
console.log(`Numele meu este: ${Nume} ${Prenume}`)
```